h2o_wave_ml/dai.py

```diff
@@ -217,7 +217,7 @@ def _build_model(cls, train_file_path: str, train_df: Optional[PandasDataFrame],
         }
 
         if model_metric != ModelMetric.AUTO:
-            if 'scorer' not in params.get('scorer', []):
+            if 'scorer' not in params.keys():
```

Comment:
Wait,

```python3
if 'scorer' not in params:
```

should be enough here no? 🤔 

```suggestion
            if 'scorer' not in params:
```

Reply:
Yup!